python-requirements-unlocked.txt

```diff
@@ -1,6 +1,6 @@
 curies
 jinja2
-oaklib
+oaklib>=0.5.20
```

Comment:
## `oaklib` goal && ODK dependency upgrade needed
@matentzn `oaklib` just got a new release `0.5.20` and it is needed for this work. 
I created an issue in ODK for this:
- https://github.com/INCATools/ontology-development-kit/issues/936

For now, I added an `oaklib` goal that installs the latest version of it, and added that as a prerequisite for the sync subclass goal. 

<details><summary>Other stuff</summary>
<p>

**About specific version numbers: no change needed I suppose**
`oaklib` is now the only version in this file where I know the specific minimum version I need for `mondo-ingeset`, and I know it because Chris just made a release that I need. Really though, we need pretty high versions for most of these dependencies here. I guess we don't need to write the specific version anyway since `pip` will always install the latest version.

</p>
</details>

Reply:
It may be some time before ODK has `oaklib >= 0.5.20`. I just made a todo in the makefile about dropping the need for `pip-oaklib` when that upgrade happens. This is resolved for now.